redis/src/main/scala/zio/redis/Output.scala

```diff
@@ -161,6 +161,14 @@ object Output {
       Either.cond(text == "+OK\r\n", (), ProtocolError(s"$text isn't unit."))
   }
 
+  case object SimpleStringOutput extends Output[String] {
```

Comment:
When I think more about this, I think we should do the following:

- Rename StringOutput to MultiStringOutput.
- Rename this to StringOutput.
- Create an ADT to represent possible redis types.
- Create a TypeOutput.

While the newly created StringOutput won't be in use, we might keep it until we verify there are no more "mistypes".